test/inductor/test_control_flow.py

```diff
@@ -2,6 +2,7 @@
 import itertools
 
 import torch
+import torch._dynamo.testing
```

Comment:
?

Reply:
This test file requires `torch._dynamo.testing` to be imported.

Somehow one of my import removals prevented the existing path, so i needed to readd it here.

https://github.com/pytorch/pytorch/blob/d133d5403bca0856429f12ab0e49c4b9c89d1bec/test/inductor/test_control_flow.py#L182